octopusdeploy/schema_action_run_kubectl_script.go

```diff
@@ -22,10 +26,29 @@ func expandRunKubectlScriptAction(flattenedAction map[string]interface{}) *deplo
 	action := expandAction(flattenedAction)
 	action.ActionType = "Octopus.KubernetesRunScript"
 
-	action.Properties["Octopus.Action.Script.ScriptFileName"] = core.NewPropertyValue(flattenedAction["script_file_name"].(string), false)
-	action.Properties["Octopus.Action.Script.ScriptParameters"] = core.NewPropertyValue(flattenedAction["script_parameters"].(string), false)
-	action.Properties["Octopus.Action.Script.ScriptSource"] = core.NewPropertyValue("Package", false)
+	if v, ok := flattenedAction["script_body"]; ok {
```

Comment:
Does the syntax property also need to be set?
Why doesn't this `expandRunScriptAction`?

Reply:
Good point, I cleaned this up.